### ISSUE TITLE
`sqlmesh plan --restate-model foo.bar --start 2024-08-01 --no-prompts` provides no feedback on what will actually be backfilled

### ISSUE THREAD

**Issue by plaflamme** [TobikoData/sqlmesh#3062]
Assuming model `foo.bar` has some downstream dependencies, it would be useful to see which models will be affected by a `restatement` plan before applying it.

**Comment by izeigerman** [comment]
@plaflamme I believe that's due to using the `--no-prompts` flag. Can you please check whether it works as expected without it?

**Comment by plaflamme** [comment]
Yes, that's correct. Ideally, when `--no-prompts` is specified, it would still print what would happen "by default".

**Comment by hardeybisey** [comment]
Hello @izeigerman, I would love to contribute to sqlmesh by working on this issue. Is there any process in place to getting an issue assigned. Thanks.

**Comment by erindru** [comment]
@hardeybisey if an issue is unassigned (like this one) and you'd like to start working on it then feel free to just start. We are always happy to accept well-formed PR's

**Comment by hardeybisey** [comment]
@erindru Thanks for the information. I’ll investigate the issue and start working on it.

**Comment by izeigerman** [comment]
Fixed in #3568 (0.142.)
